2018-11-30T10:56:14|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\ScopeFoundry\logged_quantity.py", line 241, in update_value
    self.hardware_set_func(self.val)
  File "C:\LabPrograms\Python\Camera-ScopeFoundry\CameraMeasurement.py", line 244, in setSaveH5
    self.settings.record.disable()
AttributeError: 'LoggedQuantity' object has no attribute 'disable'

